crates/holochain/src/conductor/api/api_external/app_interface.rs

```diff
@@ -226,27 +235,76 @@ pub struct AppAuthentication {
     pub installed_app_id: Option<InstalledAppId>,
 }
 
-/// Combination of zome call parameters with individual fields and the signed form which includes
-/// the serialized bytes of the parameters and the provenance's signature of the serialized bytes.
-#[derive(Clone, Debug)]
-pub struct ZomeCall {
```

Comment:
It's no longer needed to pass around signature or serialized bytes or the has of serialized bytes.